static/context.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react'
```

Comment:
Well, this _context.js_ script is a pure system code that don't seem to have its place in _static_ at all.
Wondering what made you opt for this?

Reply:
Yep so same level than the App.js